packages/material-ui/src/styles/createPalette.test.js

```diff
@@ -176,8 +176,9 @@ describe('createPalette()', () => {
           contrastThreshold: 0,
         }));
       }).toErrorDev([
-        'falls below the WCAG recommended absolute minimum contrast ratio of 3:1',
-        'falls below the WCAG recommended absolute minimum contrast ratio of 3:1',
+        'falls below the WCAG recommended absolute minimum contrast ratio of 3:1', // warning palette
+        'falls below the WCAG recommended absolute minimum contrast ratio of 3:1', // info palette
+        'falls below the WCAG recommended absolute minimum contrast ratio of 3:1', // success palette
```

Comment:
Do we include the color in the warning? If not then we can follow-up. Otherwise I'd make sure the color is included in the assertion which means we can get rid of the (potentially wrong) code comments.

Reply:
2 lines was there at the beginning (1 more is added because `info` is changed to `cyan[500]`, it's sound intentional that we expect the colors to falls below contrast ratio. I was confuse at first about this test so I added comments at the end.